kotlin/internal/jvm/compile.bzl

```diff
@@ -52,6 +52,8 @@ def _fold_jars_action(ctx, rule_kind, output_jar, input_jars):
 _CONVENTIONAL_RESOURCE_PATHS = [
     "src/main/resources",
     "src/test/resources",
+    "bazel-out/k8-fastbuild/genfiles",
```

Comment:
This seems odd. We shouldn't depend on bazel-* paths, especially if they have configuration parameters in them, like k8.